src/util/Util.js

```diff
@@ -316,6 +316,30 @@ class Util {
     }
     return f;
   }
+
+  static editPosition(item, position, sorted, route, type, reason) {
```

Comment:
Instead of creating a new method here, you can probably use `setPosition`.
Something like this could work:
```js
Util.setPosition(...).then(updatedItems =>
    this.client.actions.GuildChannelsPositionUpdate.handle({
        guild_id: this.guild.id,
        channels: updatedItems,
    });
```

Reply:
I was thinking about that too, but I did it this way so that it happens all in one function, without having to .then it and do it in the main edit function.

I will experiment with it tho.